code/datums/gamemode/role/ninja.dm

```diff
@@ -16,6 +16,34 @@
 		antag.current << sound('sound/effects/gong.ogg')
 		equip_ninja(antag.current)
 		name_ninja(antag.current)
+		
+/datum/role/ninja/proc/ThrowAtStation(var/mob/living/carbon/human/spaceninja) //copypasted from rodcode
```

Comment:
I feel the throwatstation function should be made separate from the ninja role. Maybe into misc_gamemode_procs, with its argument being an atom/movable instead of specifically a human?

Reply:
we actually have a spawns from edge proc in new_player for traders

Reply:
I was thinking that myself considering the copypaste from rods. I'd have to add a radius var as well, but seems like a good idea.

Reply:
@Kurfursten your proc, `Meteortype_Latejoin`, requires landmarks. This doesn't.

Reply:
#22929